app/models/story.rb

```diff
@@ -8,7 +8,7 @@ class Story < ActiveRecord::Base
   belongs_to :project, touch: true
   belongs_to :owner, class_name: "User"
   has_many :tasks, -> { order(completed: :asc, order: :asc, created_at: :asc) }
-  has_many :attachments
+  has_many :attachments, -> { where("attachable_id IS NOT NULL") }
```

Comment:
I've changed these to make sure the attachment was actually saved - it is currently adding attachments a user could upload in a comment, even if they never post the comment.